test/rtp/rtp_test.clj

```diff
@@ -5,19 +5,22 @@
 (deftest final-test
   (testing "RTP of some weird hands"
     (is (= (/ 1625 3)
-          (rtp/rtp [{:values [0 9 10 11 12]
-                     :suits  [0 0 0 0 0]
-                     :times  2}
-                    {:values [0 0 0 0 1]
-                     :suits  [0 1 2 3 1]
-                     :times  1}])))
-    (is (= (/ 880336 632385)
-           (rtp/rtp [{:values [0 11 10 9 8]
+           (rtp/rtp [{:values [0 9 10 11 12]
+                      :suits  [0 0 0 0 0]
+                      :times  2}
+                     {:values [0 0 0 0 1]
+                      :suits  [0 1 2 3 1]
+                      :times  1}
+                     {:values [0 9 10 11 12]
+                      :suits  [0 0 0 0 0]
+                      :times  2}
+                     {:values [0 0 0 0 1]
+                      :suits  [0 1 2 3 1]
+                      :times  1}])))
+    (is (= (/ 86698 61617)
+(time      (rtp/rtp [{:values [0 11 10 9 8]
```

Comment:
I left it here because it lets me see if future perfomance changes have any incidence.